Apps/WebClient/src/ClientApp/src/services/httpDelegate.ts

```diff
@@ -132,18 +132,6 @@ export default class HttpDelegate implements IHttpDelegate {
                     this.logger.error(errorMessage);
                     return reject(errorMessage);
                 });
-
-            // TODO: Axios has bug with the delete method not using data fields.
```

Comment:
Todo needs to remain until we it is resolved

Reply:
https://qslvic.visualstudio.com/Health%20Gateway/_workitems/edit/10506